.snippets/code/builders/ethereum/json-rpc/debug-trace/terminal/start-up-logs.md

```diff
@@ -1,7 +1,7 @@
 <div id="termynal" data-termynal>
   <span data-ty="input"><span class="file-path"></span>docker run --network host  \
     <br>-u $(id -u ${USER}):$(id -g ${USER}) \
-        moonbeamfoundation/moonbeam-tracing:v0.38.0-3000-latest \
+        moonbeamfoundation/moonbeam-tracing:v0.39.0-3000-latest \
```

Comment:
```suggestion
        moonbeamfoundation/moonbeam-tracing:v0.39.0-3100-latest \
```

Tracing tags are now on 3100 latest, you can double-check this here: https://hub.docker.com/r/moonbeamfoundation/moonbeam-tracing/tags